Vasili Makouchyk/1/database.rb

```diff
@@ -0,0 +1,39 @@
+class DataBase
+  require 'nokogiri'
+  require 'mechanize'
+  require 'redis'
+  require 'open-uri'
+
+  URL = 'http://rapstyle.su/quotes.php'.freeze
+  FILE = 'citaty.html'.freeze
+
+  def initialize
+    @database = Redis.new
+    @quotes = []
+    download_page
```

Comment:
You're breaking single responsibility principle.
This class takes care of both presistence and scraping.
Ideal option is to make two separate classes.
It would be enough to move scraping into separate method:
```
def initialize
  @database = Redis.new
end

def load_quotes
  ....find quotes, persist....
  self
end
```
So in bot class you could write something like:
```@database = Database.new.load_quotes```